Source/VCGeneration/ConditionGeneration.cs

```diff
@@ -1421,6 +1421,12 @@ private void AddDebugInfo(Cmd c, Dictionary<Variable, Expr> incarnationMap, List
       else if (c is CommentCmd)
       {
         // comments are just for debugging and don't affect verification
+      } else if (c is HideRevealCmd)
+      {
+        passiveCmds.Add(c);
```

Comment:
The fact that these stick around until the passive stage is unfortunate, in the sense that it's nice for that set of commands to be as small as possible, but I think it's also necessary for the goal of this PR.

Reply:
Seems fine to me :-) They are used only in the passive stage.